### ISSUE TITLE
Add a Windows-like themes/icons

### ISSUE THREAD

**Issue by 2br-2b** [Opticos/GWSL-Source#112]
Right now, the default themes on WSL are great, but they conflict with my Windows desktop. The colors, button shapes, icons, etc. clash with the rest of my desktop:

![image](https://user-images.githubusercontent.com/43968991/141854736-54ccc9f5-0749-4c1a-9547-70303dbf109e.png)

It'd be nice if GWSL automatically installed and activated a windows-like GTK and icon theme to better match the rest of my desktop.

I'm sure there are plenty of these themes out there - the question is finding one with a good, usable license.

I would have suggested B00merang Projects [light](https://github.com/B00merang-Project/Windows-10) and [dark](https://github.com/B00merang-Project/Windows-10-Dark) themes, but they're under the GPLv3, so ~~I assume~~ you'd have to change GWSL's license to include them.

**Comment by 2br-2b** [comment]
https://www.gnome-look.org/p/1387736/ could be a good icon pack for Win 10. As a note, however, there is an [XSS vulnerability](https://positive.security/blog/hacking-linux-marketplaces) which, to the best of my knowledge, hasn't been patched. I have yet to test it, but since it's been 2 years since Pling's repos were last updated, I'd assume the vulnerability is still there.

On the other hand, I would assume it'd be okay to directly copy Windows' files. Copyright shouldn't be an issue since the files are already on users' computers.

**Comment by Pololot64** [comment]
First of all the files are in the wrong format and would have to be messed with alot. The B00merang themes are great (And I do not believe I would have to change my license I think...) but I do not want GWSL messing with wsl distros too much. Currently I only edit the shell config files. However, GWSL does have a theming menu to set gtk themes and it will automatically launch programs with them. Simply installing the theme (and icons) oneself would be best. Otherwise gwsl would be messing around too much.

**Comment by 2br-2b** [comment]
Fair enough. Maybe make WSL download the themes, but not apply them unless the user specifically asks for them to be activated?

> GWSL does have a theming menu to set gtk themes and it will automatically launch programs with them.

Maybe there could be a button which downloads and adds the themes to the .themes folders?

**Comment by Pololot64** [comment]
We shall see :)
